docs/spec.md

```diff
@@ -1136,34 +1149,40 @@ def batch_norm_inference(operand, scale, offset, mean, variance, epsilon, featur
 ```
 
+For quantized types, performs
+`dequantize_op_quantize(lambda operand, scale, offset, mean, variance:
+batch_norm_inference(operand, scale, offset, mean, variance, epsilon,
+feature_index), operand, scale, offset, mean, variance, type(result))`.
+
 #### Inputs
 
-| Label | Name            | Type                                        | Constraints   |
-|-------|-----------------|---------------------------------------------|---------------|
-| (I1)  | `operand`       | tensor of floating-point type               | (C1-C7)       |
-| (I2)  | `scale`         | 1-dimensional tensor of floating-point type | (C2), (C3)    |
-| (I3)  | `offset`        | 1-dimensional tensor of floating-point type | (C2), (C4)    |
-| (I4)  | `mean`          | 1-dimensional tensor of floating-point type | (C5)          |
-| (I5)  | `variance`      | 1-dimensional tensor of floating-point type | (C2), (C6)    |
-| (I6)  | `epsilon`       | constant of type `f32`                      |               |
-| (I7)  | `feature_index` | constant of type `si64`                     | (C1), (C3-C6) |
+| Label | Name            | Type                                                                | Constraints   |
+|-------|-----------------|---------------------------------------------------------------------|---------------|
+| (I1)  | `operand`       | tensor of floating-point type or per-tensor quantized tensor        | (C1-C7)       |
+| (I2)  | `scale`         | 1-dimensional tensor of floating-point or per-tensor quantized type | (C2), (C3)    |
+| (I3)  | `offset`        | 1-dimensional tensor of floating-point or per-tensor quantized type | (C2), (C4)    |
+| (I4)  | `mean`          | 1-dimensional tensor of floating-point or per-tensor quantized type | (C5)          |
+| (I5)  | `variance`      | 1-dimensional tensor of floating-point or per-tensor quantized type | (C2), (C6)    |
+| (I6)  | `epsilon`       | constant of type `f32`                                              |               |
+| (I7)  | `feature_index` | constant of type `si64`                                             | (C1), (C3-C6) |
 
 #### Outputs
 
-| Name     | Type                          | Constraints |
-|----------|-------------------------------|-------------|
-| `result` | tensor of floating-point type | (C2), (C7)  |
+| Name     | Type                                                         | Constraints |
+|----------|--------------------------------------------------------------|-------------|
+| `result` | tensor of floating-point type or per-tensor quantized tensor | (C2), (C7)  |
 
 #### Constraints
 
 * (C1) `0 <= feature_index < rank(operand)`.
-* (C2) `operand`, `scale`, `offset`, `mean`, `variance` and `result` have the
-  same element type.
+* (C2) `baseline_element_type(operand) = baseline_element_type(scale) =
+       baseline_element_type(offset) = baseline_element_type(mean) =
+       baseline_element_type(variance) = baseline_type(result)`.
```

Comment:
nit: shouldn't this be baseline_element_type(result) instead of baseline_type(result)?